presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -873,6 +873,20 @@ public void testDistinctHaving()
     }
 
     @Test
+    public void testSelectFilter()
+            throws Exception
+    {
+        assertQuery("SELECT sum(x) FILTER (WHERE y > 4) FROM (VALUES (1, 3),(2, 4),(2, 4),(4, 5)) t (x, y)", "SELECT 4");
+        assertQuery("SELECT sum(x) FILTER (WHERE x > 1), sum(y) FILTER (WHERE y > 4) FROM (VALUES (1, 3),(2, 4),(2, 4),(4, 5)) t (x, y)", "SELECT 8, 5");
+        assertQuery("SELECT sum(x) FILTER(WHERE x > 1), sum(x) FROM (VALUES (1),(2),(2),(4)) t (x)", "SELECT 8, 9");
```

Comment:
Missing space after `FILTER` and after commas. Please fix this for all queries in this method.